sinks/api/openapi.yaml

```diff
@@ -139,6 +139,34 @@ paths:
           description: A non-existent entity request.
         '500':
           $ref: "#/components/responses/ServiceErrorRes"
+  /sinks/validate:
```

Comment:
we also need to remove `validate_only` from `SinkCreateReqSchema` and fix the description on createsSink to no longer reference pre validation